src/maps/ireland/rules.tsx

```diff
@@ -1,6 +1,6 @@
+import {ReactNode} from "react";
 
-export function IrelandRules() {
-  return <div>
+export const RULES: ReactNode = <div>
```

Comment:
Is this a common practice in React? Does the factory for a ReactNode carry any kind of performance impact that might impact startup performance?

Reply:
Not particularly; I was doing it this way to avoid having to change all the `settings.ts` files over to `tsx`. I think there's a better way to achieve that though without exposing these nodes as `consts` now that I think about it though. Let me get an update in for that.

Reply:
Ok, just pushed an update that has the rules files as proper react components instead, and then in `settings.ts` uses `React.createElement(FooRules)` in lieu of changing those files over to full tsx.

Reply:
Could you just do getMapSettings = CyprusRules;

Reply:
Oh yea, you can do that! Much better!